apps/myjobs/app/views/workflows/show.json.jbuilder

```diff
@@ -3,16 +3,17 @@ json.set! 'status_label', status_label(@workflow)
 json.set! 'active', @workflow.active?
 json.set! 'fs_root', Filesystem.new.fs(@workflow.staged_dir)
 json.set! 'host_title', cluster_title(@workflow.batch_host)
-json.folder_contents (@workflow.folder_contents) do |content|
-  json.set! 'path', content
-  json.set! 'name', Pathname(content).basename.to_s
-  json.set! 'type', Pathname(content).file? ? 'file' : 'dir'
-  json.set! 'fsurl', Filesystem.new.fs(content)
-  json.set! 'fs_base', Filesystem.new.fs(File.dirname(content))
+json.folder_contents (@workflow.folder_contents).each do |content|
```

Comment:
Since `folder_contents` return `WorkflowFile` objects, maybe the variable `content` should be replaced by `file`, then `file.path` reads better than `content.path`.